apps/web/src/app/auth/auth.module.css

```
@@ -0,0 +1,217 @@
+  font-weight: 600;
+  color: #5f6b7a;
+  font-size: 0.85rem;
+  text-transform: uppercase;
+  letter-spacing: 0.06em;
+  white-space: nowrap;
+}
+
+.rowValue {
+  color: #111827;
+  font-weight: 500;
+  word-break: break-word;
```

Stylelint (17.6.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)